csharp/api-cors-lambda-crud-dynamodb/src/ApiCorsLambdaCrudDynamodb/ApiCorsLambdaCrudDynamodbStack.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections.Generic;
+using Amazon.CDK;
+using Amazon.CDK.AWS.APIGateway;
+using Amazon.CDK.AWS.DynamoDB;
+using Amazon.CDK.AWS.Lambda;
+
+namespace ApiCorsLambdaCrudDynamodb
+{
+    public class ApiCorsLambdaCrudDynamodbStack : Stack
+    {
+        public ApiCorsLambdaCrudDynamodbStack(Construct parent, string id, IStackProps props) : base(parent, id, props)
+        {
+            var dynamoTable = new Table(this, "items", new TableProps{
+                PartitionKey = new Attribute{
+                    Name = "itemId",
+                    Type = AttributeType.STRING
+                },
+                TableName = "Items",
+
+                // The default removal policy is RETAIN, which means that cdk destroy will not attempt to delete
+                // the new table, and it will remain in your account until manually deleted. By setting the policy to 
+                // DESTROY, cdk destroy will delete the table (even if it has data in it)
```

Comment:
it's CloudFormation that will skip deployment / delete based on these properties. Not quite `cdk destroy` that takes the action. Might be worth rewording for users that work with the template outside of the `CLI`